File: go/iq/md~set~companion_identity_public+link_code_companion_reg+link_code_pairing_ref+link_cod-707b2467.go
// BuildMdCompanionFinish builds <iq type="set" xmlns="md">. Replies with one of: Success, Error.
func BuildMdCompanionFinish(jid types.JID) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"xmlns": "md", "to": "s.whatsapp.net", "type": "set"},
		Content: []waBinary.Node{
			{Tag: "link_code_companion_reg", Attrs: waBinary.Attrs{"jid": jid, "stage": "companion_finish"}},
		},
	}
}

File: ts/iq/md~set~companion_identity_public+link_code_companion_reg+link_code_pairing_ref+link_cod-707b2467.ts
/** <iq type="set" xmlns="md">. Replies with one of: Success, Error. */
export function buildMdCompanionFinish(jid: string): BinaryNode {
  return { tag: 'iq', attrs: { xmlns: 'md', to: 's.whatsapp.net', type: 'set' }, content: [
      { tag: 'link_code_companion_reg', attrs: { jid: jid, stage: 'companion_finish' } },
    ] }
}